src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Components/AvailableResult.cs

```diff
@@ -20,7 +20,7 @@ internal class AvailableResult
         internal string Label { get; set; }
 
         /// <summary>
-        /// Gets or sets an an alternative search tag that will be evaluated if label doesn't match. For example we like to show the era on searches for 'year' too.
+        /// Gets or sets an alternative search tag that will be evaluated if label doesn't match. For example we like to show the era on searches for 'year' too.
```

Comment:
Note that this fix is because of the rule against two articles in a row. check-spelling's rule against doubled words is tuned to words of at least 3 characters which is why this snuck in.